src/App.jsx

```diff
@@ -3,16 +3,17 @@ import Footer from "./components/Footer";
 import Header from "./components/Header";
 import "./App.css";
 import { Reset } from "styled-reset";
+import StationsList from "./components/StationsList";
 
 function App() {
-  return (
-    <div>
-      <Reset />
-      <Header />
-      <MapLille />
-      <Footer />
-    </div>
-  );
+	return (
+		<div>
+			<Reset />
+			<Header />
+			<StationsList />
+			<Footer />
+		</div>
+	);
```

Comment:
MapLille component disappeared, you forgot to use React router for keep initials components and make a redirection of <StationsList/> into <MapLille/> trough index.jsx in my opinion.